WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -106,16 +106,31 @@ class StoreOnboardingViewModel: ObservableObject {
             return
         }
 
+        analytics.track(event: .DynamicDashboard.cardLoadingStarted(type: .onboarding))
         await update(state: .loading)
-        if let tasks = try? await loadTasks(),
-           tasks.isNotEmpty {
+
+        var tasks: [StoreOnboardingTaskViewModel] = []
```

Comment:
Super nit: To prevent `tasks` being written into more than once, I would use a `let` here and set it as an empty array after catching an error.

Reply:
Good point! Updated in 0411458253fca975c197d354ce40a9a048bf96ef.